functionaltests/src/test/java/util/constant/WebDriverCommon.java

```diff
@@ -5,7 +5,7 @@
  */
 public final class WebDriverCommon {
 
-    public static final String CSS_SELECTOR_SUBMIT = "button[type=submit]";
+    public static final String XPATH_SUBMIT = "//button[@type='submit']";
```

Comment:
correct, we would need additional logic for such a case (for instance, select by ID, name or class), but it is very unexpected right now that sofortüberweisung will have such a design.